vspace/src/main/webapp/WEB-INF/web.xml

```diff
@@ -37,21 +39,34 @@
 	<!-- spring security -->
 	<filter>
 		<filter-name>springSecurityFilterChain</filter-name>
-		<filter-class>org.springframework.web.filter.DelegatingFilterProxy</filter-class>
+		<filter-class>org.springframework.web.filter.DelegatingFilterProxy
+		</filter-class>
 	</filter>
 
 	<filter-mapping>
 		<filter-name>springSecurityFilterChain</filter-name>
 		<url-pattern>/*</url-pattern>
 	</filter-mapping>
 
+	<filter>
+		<filter-name>SpringOpenEntityManagerInViewFilter</filter-name>
+		<filter-class>org.springframework.orm.jpa.support.OpenEntityManagerInViewFilter
+		</filter-class>
+	</filter>
+	<filter-mapping>
+		<filter-name>SpringOpenEntityManagerInViewFilter</filter-name>
+		<url-pattern>/*</url-pattern>
+	</filter-mapping>
```

Comment:
what does this do?

Reply:
[VSPC-4] I get this error when I try to save an Exhibition: 
org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: edu.asu.diging.vspace.core.model.impl.Space.spaceLinks, could not initialize proxy - no Session

Since the Exhibition is getting the space and the space links are lazy loaded in the SpaceManager along with it, this is happening. Is there a better way to resolve this issue?

Reply:
No, this is actually pretty cool. I didn't know this existed. Do you understand what the issue was and how you solved it?

Reply:
Yes. I worked on lazy loading a bit before. But now I clearly understood. I read an article that explained what causes this error and how lazy loading is used. 

One article said using this open entity manager may setback performance depending on how it's used.